src/main/docs/guide/appendix/breaks.adoc

```diff
@@ -6,6 +6,14 @@ In Micronaut 1.x the ann:context.annotation.Executable[] annotation was a meta-a
 
 In Micronaut 2.x and above this is no longer the case and if you need an api:inject.ExecutableMethod[] generated you should add the ann:context.annotation.Executable[] annotation to any custom AOP advice you specify otherwise the method will not be available via the api:inject.BeanDefinition[] interface (using for example the api:BeanDefinition.getExecutableMethods[] method).
 
+=== Spot Bugs Instead of JSR-305 Nullable/NonNull Annotations
```

Comment:
I would put this under whats new and just add a blurb here about how the dependency isn't part of the API of core any longer